tools/schemacode/src/bidsschematools/render/text.py

```diff
@@ -183,7 +178,9 @@ def make_glossary(schema, src_path=None):
             levels = [level["name"] if isinstance(level, dict) else level for level in levels]
             text += f"**Allowed values**: `{'`, `'.join(levels)}`\n\n"
 
-        text += f"**Description**:\n{obj_desc}\n\n"
+        # Convert description into markdown and append to text
+        obj_desc = MarkdownIt().render(f"**Description**:\n{obj_desc}")
```

Comment:
@Remi-Gau the pdf build doesn't like this

Reply:
Description is still gone for the PDF. The problem is that other renderers are aware of whether the PDF or the Markdown versions are to be rendered. However, here we do not know that information.

If this is not Markdowned, then the HTML version doesn't render well the ` ``` ` blocks. If Markdowned, it's missing from the PDF.

Reply:
fine with me: in any case the schema rendering code needs a big refactor IMO and with probably some jinja-ification of the whole thing

Reply:
It wasn't clear at all, I would've saved you time with just a few more words ;)